tests/samples/files/handlebarsjs_filter_attr_value_style_002.hbs

```diff
@@ -6,3 +6,6 @@
 <div style ="   {{style4}}  ">
 <div style ='   {{style5}}  '>
 <div style =    {{style6}}  >
+<div style ="&#0*32;&#x20;&#9;&#x09;&Tab;&#10;&#x0a;&NewLine;&#12;&#x0c;&#13;&#x0d;{{style7}}&#0*32;  ">
+<div style ='&#0*32;&#x20;&#9;&#x09;&Tab;&#10;&#x0a;&NewLine;&#12;&#x0c;&#13;&#x0d;{{style8}}&#0*32;  '>
+<div style =&#0*32;&#x20;&#9;&#x09;&Tab;&#10;&#x0a;&NewLine;&#12;&#x0c;&#13;&#x0d;{{style9}}&#0*32;    >
```

Comment:
yes the zero is added intentionally. however, those cases are not needed anymore, as i have another set of unit test to cover.

but we need to add back one test case to catch the parse error when there is ';' left after the html decoding.